lib/matplotlib/tests/test_figure.py

```diff
@@ -92,13 +92,22 @@ def test_gca():
     assert_true(fig.gca() is ax1)
 
 
-@image_comparison(baseline_images=['figure_suptitle'])
+@image_comparison(baseline_images=['figure_suptitle',
+                                   'figure_suptitle_fontprops'])
 def test_suptitle():
     fig = plt.figure()
     ax = fig.add_subplot(1, 1, 1)
     fig.suptitle('hello', color='r')
     fig.suptitle('title', color='g', rotation='30')
 
+    from matplotlib.font_manager import FontProperties
```

Comment:
Can you make this it's own test with the `@cleanup` decorator?  Just checking that font size and weight are correct is probably enough for this test.